test_utils/fake_catalog_api.py

```diff
@@ -1342,6 +1343,18 @@ def get_fake_catalog_diff_create_w_program():
     ], [], []
 
 
+def get_fake_catalog_diff_w_one_each():
+    """
+    Returns a fake response from the EnterpriseCatalogApiClient.get_catalog_diff where two courses and a program are
+    returned, one in each change-type bucket
+    """
+    # items_to_create, items_to_delete, matched_items
```

Comment:
cruft?

Reply:
it was in other similar methods, but i can clean it up